src/components/buttons/ButtonSecondary.jsx

```diff
@@ -30,7 +30,6 @@ ButtonSecondary.propTypes = {
   wide: PropTypes.bool,
   disabled: PropTypes.bool,
   children: PropTypes.node.isRequired,
-  icon: PropTypes.node,
```

Comment:
FYI this wasn't used so we removed it - it's not connected with other changes in this PR